cluster-autoscaler/README.md

```diff
@@ -1,4 +1,80 @@
-Cluster Autoscaler automatically adjusts the number of nodes in a cluster.
+# Cluster Autoscaler
 
-**Status:** under development
+# Introduction
 
+Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernetes cluster when:
+* there is a pod that doesn’t have enough space to run in the cluster
+* some nodes in the cluster are so underutilized that they can be deleted and their pods will 
+be easily placed on some other, existing nodes.  
+
+# Deployment
+
+Cluster Autoscaler runs on the Kubernetes master node (at least in the default setup on GCE and GKE). 
+It is possible to run customized Cluster Autoscaler inside of the cluster but then extra care needs
+to be taken to ensure that Cluster Autoscaler is up and running. User can put it into kube-system
+namespace (Cluster Autoscaler doesn't scale down node with non-manifest based kube-system pods running
+on them) and mark with `scheduler.alpha.kubernetes.io/critical-pod` annotation (so that the rescheduler, 
+if enabled, will kill other pods to make space for it to run). 
+
+Right now it is possible to run Cluster Autoscaler on:
+* GCE http://kubernetes.io/docs/admin/cluster-management/#cluster-autoscaling
+* GKE https://cloud.google.com/container-engine/docs/cluster-autoscaler
+* AWS https://github.com/kubernetes/contrib/blob/master/cluster-autoscaler/cloudprovider/aws/README.md
+
+# Scale Up
+
+Scale up creates a watch on the api server looking for all pods. Every 10 seconds (configurable)
```

Comment:
Than I think that this document is still not addressing this concern. I think you'd have to explicitly explain what's taking so long.